guide/06-imagery-and-raster-analysis/lidar_basics_part1.ipynb

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
<ul style="user-select: auto;"><li style="user-select: auto;"><strong style="user-select: auto;">2nd paragraph</strong>: These receivers .... the range <strong style="user-select: auto;">(</strong>distance<strong style="user-select: auto;">)</strong> ... (put distance within parenthesis)</li></ul><p style="user-select: auto;"><br style="user-select: auto;"></p><ul style="user-select: auto;"><li style="user-select: auto;"><strong style="user-select: auto;">First return</strong>: ...user often gets... -&gt; user<strong style="user-select: auto;">s </strong>often get<s style="user-select: auto;">s</s> exposed.</li><li style="user-select: auto;">One emitted laser pulse -&gt; A single emitted laser...</li><li style="user-select: auto;">that encounters multiple reflection surfaces -&gt; that encounters multiple <strong>reflective </strong>surfaces...</li></ul><p style="user-select: auto;"><br style="user-select: auto;"></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/610/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>